testing/trino-tests/src/test/java/io/trino/execution/resourcegroups/db/TestQueuesDb.java

```diff
@@ -369,6 +371,29 @@ public void testNonLeafGroup()
         assertEquals(queryRunner.getCoordinator().getDispatchManager().getQueryInfo(invalidResourceGroupQuery).getErrorCode(), INVALID_RESOURCE_GROUP.toErrorCode());
     }
 
+    @Test
+    public void testUpdateSoftMemoryLimit()
+            throws Exception
+    {
+        // trigger resource group creation
+        queryRunner.execute("SELECT COUNT(*), clerk FROM orders GROUP BY clerk");
+        InternalResourceGroupManager<?> manager = queryRunner.getCoordinator().getResourceGroupManager().get();
+        DbResourceGroupConfigurationManager dbConfigurationManager = (DbResourceGroupConfigurationManager) manager.getConfigurationManager();
+
+        dao.updateResourceGroup(2, "bi-${USER}", "1MB", 3, 2, 2, null, null, null, null, null, 1L, TEST_ENVIRONMENT);
+        dbConfigurationManager.load();
```

Comment:
is this "required" for the test? AFAIK we only care about switching from fraction based to exact value so whatever was used before fraction based doesn't seem to matter.

Reply:
Many thanks for the comments !

> is this "required" for the test?
We don't need that actually, will get rid of this line.